x/stake/client/cli/tx.go

```diff
@@ -23,7 +24,7 @@ func NewStakeTxCmd() *cobra.Command {
 		Long: strings.TrimSpace(
 			fmt.Sprintf(`Stake on a post.
 Example:
-$ %s tx stake stake 1 "2" 500 --from mykey
+$ %s tx stake stake 1 "2" 500 starsvaloper1deadbeef --from mykey
```

Comment:
:joy:

Reply:
Hah, any opportunity to use a magic debug value https://en.wikipedia.org/wiki/Magic_number_(programming)#Debug_values.